### ISSUE TITLE
click while blinking issue

### ISSUE THREAD

**Issue by screwyluie** [pfn/passifox#490]
often, when I change a password and chromeipass detects it, the icon blinks as it should but when I click on the icon it just stops blinking. No dialog to change/create the key is shown, it just disables the blinking and I'm left to manually do it. Thank god for copy to clipboard button or I'd be screwed.

This happen often, I feel it's about a 50/50 chance

chrome 49.0.2623.112

**Comment by zeromido** [comment]
Try changing "Blink Time" and "Redirect Offset" values and note that it's in ms 
(60000 millisecond = 1 second)

**Comment by screwyluie** [comment]
That was the first thing I did, changed it so it would blink for twenty seconds. Doesn't change the issue, it still doesn't do anything most of the time.

**Comment by zeromido** [comment]
Hmm... Try reinstalling [chromeIPass](https://chrome.google.com/webstore/detail/chromeipass/ompiailgknfdndiefoaoiligalphfdae)

**Comment by zeromido** [comment]
Did it work?

**Comment by screwyluie** [comment]
Nope. Since then I've reinstalled everything, including windows, have this issue

**Comment by screwyluie** [comment]
*still have this issue

**Comment by zeromido** [comment]
Okay. Try this as a last resort
Download https://github.com/pfn/passifox/archive/master.zip and extract it then open chromeIPass folder and copy it's path.
Go to chrome://extensions/ and check "Developer mode" then click "Load unpacked extension" and paste the path you copied and click OK.

**Comment by zeromido** [comment]
What happened? Did it work?

**Comment by screwyluie** [comment]
forgive me, I've been super busy lately.

Failed to load extension from: D:\passifox-master\passifox
Manifest file is missing or unreadable.

the manifest is in there, but chrome doesn't want to load it.

**Comment by zeromido** [comment]
passifox!? You mean ChromeIPass?

**Comment by screwyluie** [comment]
Your link above goes to passifox, so that's what I used. There's a chrome folder in there but it doesn't have a manifest

**Comment by zeromido** [comment]
How's that? It does have a manifest.
http://i.imgur.com/741s0Lw.png 
FYI passifox is for Firefox. So you need to load chromeipass folder.
If it didn't work try installing the new chromeipass version https://chrome.google.com/webstore/detail/chromeipass/ompiailgknfdndiefoaoiligalphfdae

**Comment by screwyluie** [comment]
ok well I don't know what went wrong there, the first one wouldn't load even if I pointed it to the chrome folder. I downloaded it again and now it works. Sorry for the confusion, I'll will not begin testing it

**Comment by gab** [comment]
I've been having the exact same issue for a long time (several months, perhaps 6 months or more). The save/update password feature has become so unreliable that I just enter passwords manually in the database now.

 Chrome 53.0.2785.143 m, chromeipass installed from web store
